ui/components/app/advanced-gas-fee-popover/advanced-gas-fee-inputs/base-fee-input/base-fee-input.js

```diff
@@ -105,25 +105,61 @@ const BaseFeeInput = () => {
   const [editingInGwei, setEditingInGwei] = useState(false);
 
   const [maxBaseFeeGWEI, setMaxBaseFeeGWEI] = useState(() => {
-    if (
-      estimateUsed !== PRIORITY_LEVELS.CUSTOM &&
-      advancedGasFeeValues?.maxBaseFee
+    if (estimateUsed !== PRIORITY_LEVELS.CUSTOM) {
+      if (advancedGasFeeValues?.maxBaseFeeGWEI) {
+        return advancedGasFeeValues.maxBaseFeeGWEI;
+      } else if (advancedGasFeeValues?.maxBaseFeeMultiplier) {
+        return multiplyCurrencyValues(
+          estimatedBaseFee,
+          advancedGasFeeValues.maxBaseFeeMultiplier,
+          numberOfDecimalsPrimary,
+        );
+      }
+    } else if (
+      advancedGasFeeValues?.maxBaseFeeGWEI &&
+      advancedGasFeeValues?.maxBaseFeeGWEI === maxFeePerGas
+    ) {
+      return advancedGasFeeValues.maxBaseFeeGWEI;
+    } else if (
+      advancedGasFeeValues?.maxBaseFeeMultiplier &&
+      advancedGasFeeValues?.maxBaseFeeMultiplier === maxFeePerGas
     ) {
       return multiplyCurrencyValues(
         estimatedBaseFee,
-        advancedGasFeeValues.maxBaseFee,
+        advancedGasFeeValues.maxBaseFeeMultiplier,
         numberOfDecimalsPrimary,
       );
     }
     return maxFeePerGas;
   });
 
   const [maxBaseFeeMultiplier, setMaxBaseFeeMultiplier] = useState(() => {
-    if (
-      estimateUsed !== PRIORITY_LEVELS.CUSTOM &&
-      advancedGasFeeValues?.maxBaseFee
+    if (estimateUsed !== PRIORITY_LEVELS.CUSTOM) {
+      if (advancedGasFeeValues) {
+        if (advancedGasFeeValues.maxBaseFeeMultiplier) {
+          return advancedGasFeeValues.maxBaseFeeMultiplier;
+        } else if (advancedGasFeeValues.maxBaseFeeGWEI) {
+          return divideCurrencyValues(
+            advancedGasFeeValues.maxBaseFeeGWEI,
+            estimatedBaseFee,
+          );
+        }
+      }
+    } else if (
+      Boolean(advancedGasFeeValues) &&
+      advancedGasFeeValues.maxBaseFeeMultiplier &&
+      advancedGasFeeValues.maxBaseFeeMultiplier === maxFeePerGas
+    ) {
```

Comment:
Same as this https://github.com/MetaMask/metamask-extension/pull/13099/files#r778012701